ddtrace/contrib/botocore/patch.py

```diff
@@ -20,6 +23,41 @@
 ARGS_NAME = ('action', 'params', 'path', 'verb')
 TRACED_ARGS = ['params', 'path', 'verb']
 
+propagator = HTTPPropagator()
+
+
+def modify_client_context(client_context_base64, trace_headers):
+    client_context_json = base64.b64decode(client_context_base64).decode('utf-8')
+    client_context_object = json.loads(client_context_json)
+
+    if 'Custom' in client_context_object:
+        client_context_object['Custom']['_datadog'] = trace_headers
+    else:
+        client_context_object['Custom'] = {
+            '_datadog': trace_headers
+        }
+
+    return base64.b64encode(json.dumps(client_context_object).encode('utf-8')).decode('utf-8')
+
+
+def inject_trace_to_client_context(args, span):
+    trace_headers = {}
+    propagator.inject(span.context, trace_headers)
+
+    params = args[1]
```

Comment:
Could params also be passed as a keyword argument?

Reply:
I think it's a positional argument https://github.com/boto/botocore/blob/develop/botocore/client.py#L631 but I might not understand you question here

Reply:
:+1: yeah we sometimes have it where you can pass an argument positionally (it ends up in `args`) or as a keyword argument (ends up in `kwargs`). Looks like we're ok here though :+1: